python/test/hail/methods/test_impex.py

```diff
@@ -166,6 +166,11 @@ def test_export_vcf(self):
         dataset_imported = hl.import_vcf('/tmp/sample.vcf')
         self.assertTrue(dataset._same(dataset_imported))
 
+        no_sample_dataset = dataset.filter_cols(False)
```

Comment:
I think you also need to select_entries to make this a good test.

Also what happens if you have no entry fields but you do have samples? 🤔

Reply:
I'll add select_entries(), but I think the behavior if samples and no entry fields is beyond the scope of this PR?

Reply:
yeah, totally. I'm just wondering.